### PR TITLE
Add/update README.md with project overview and instructions

### PR DESCRIPTION
**Pull request by nikneural**
This commit updates the README.md file to provide a comprehensive overview of the project and its objectives. It includes information on the project's structure, dependencies, examples.

Changes made:
- Added project overview
- Added instructions for running the project
- Added section on contributing to the project
- Added example for working with the project

Fixes #1

### CODE REVIEW

File: README.md
Comment (uberkinder) [inline README.md:1]: Code should be much simpler and should not require non-nessessary libraries (lgbm, catboost)

Comment (uberkinder) [inline README.md:1]: Don't understand why we have a link to https://pypi.org/project/pyportfolioopt/ below the image
Image is not fit the project

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Data analysis and manipulation
+numpy~=1.21.5
+pandas~=1.3.5
+
+# Testing
+pytest~=7.3.1
+
+# Machine learning
+scikit-learn~=1.0.1
+lightgbm~=3.3.5
```
Comment (uberkinder) [inline requirements.txt:10]: Should lgbm, catboost, xgboost be prerequisits?

Comment (nikneural) [reply requirements.txt:10]: Yeah, because we use this libraries in crossval._curve.py file

File: README.md
Comment (nikneural) [reply README.md:1]: Remove example from README.md.
I will add new example later!

**Review by uberkinder — APPROVED** [review]

**Pull request merged**